format/index.md

```diff
@@ -592,16 +602,6 @@ The following characters **MUST NOT** be used in member names:
 Data, including resources and relationships, can be fetched by sending a
 `GET` request to an endpoint.
 
-Clients **MUST** indicate that they can accept the JSON API media type, per
-the semantics of the HTTP `Accept` header. If present, this header value **MUST**
-also include any media type extensions relevant to the request. Servers **MUST**
-return a `406 Not Acceptable` status code if this header specifies an
```

Comment:
I removed this line about "Servers **MUST** return a `406 Not Acceptable` status code if this header specifies an unsupported media type" because it seems unnecessarily strict. 

The HTTP spec explicitly says that if "none of the available representations for the response have a media type that is listed as acceptable, the origin server can either honor the header field by sending a 406 (Not Acceptable) response or disregard the header field by treating the response as if it is not subject to content negotiation.” Why should we take away this freedom to return some representation, which might be better than an unhelpful 406, if HTTP explicitly allows it? 

Again, that fallback representation couldn't be the JSON API media type because of the new line I mentioned earlier: "Servers MUST NOT send responses with the application/vnd.api+json media type if the client has not indicated...it can understand this format." So removing this line leaves us the guarantees we want, but keeps us out of specifying things that are unnecessary and/or beyond the prerogative of the spec.